cosmoplots/colors.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from colour import Color
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib.pyplot as plt
+from matplotlib import cm, colors
+from typing import List
+
+
+def make_color_swatch(color_list: list):
+    """
+    Generate plot of the desired color swatches. This is useful if you want to see what the swatches look like.
+    This function is used in generate_hex_colors() where the plot of the swatches will show by default if used in Jupyter Notebook.
+
+    Args:
```

Comment:
Since numpy style is used elsewhere, maybe change from

```
Args:
```

to

```
Parameters
----------
```

and similar for the rest.

Reply:
I've adjusted the style, how is it now?